### PR TITLE
	ElLeeSin: 1.0.1.4 - Fixed Spell-Detection, Removed Useless variables, Fixed Multiple Ward-Casting Bug.

### PR DESCRIPTION
**Pull request by nabbhacker**

### CODE REVIEW

File: ElLeeSin/ElLeeSin/Program.cs [hunk 1]
```diff
@@ -459,7 +449,7 @@ private static void Combo()
                     || target.HasBuffOfType(BuffType.Knockback) && !Player.IsValidTarget(300)
                     && !spells[Spells.R].IsReady() || !target.IsValidTarget(Orbwalking.GetRealAutoAttackRange(Player))
                     || spells[Spells.Q].GetDamage(target, 1) > target.Health
-                    || ReturnQBuff().Distance(target) < Player.Distance(target)
+                    || ReturnQBuff()?.Distance(target) < Player.Distance(target)
```
Comment (ChewyMoon) [inline ElLeeSin/ElLeeSin/Program.cs]: C# 6??? NONONO

**Comment by nabbhacker** [comment]
..

**Comment by nabbhacker** [comment]
cmon guys at least try to accept it. Poor C#6

**Comment by myo** [comment]
you're responsable if this doesnt work

**Pull request merged**